### DISCUSSION TITLE
开始菜单字体太小且应用分类冗余

### DISCUSSION THREAD

**Discussion by ankium** [linuxdeepin/developer-center#4484] [General | 常规讨论]
![avatar](https://storage.deepin.org/thread/202305232102208137_%E5%BE%AE%E4%BF%A1%E6%88%AA%E5%9B%BE_20230523204158.png)
如图，V23 Beta版本下开始菜单的文字太小，明显和系统字体大小设置不匹配！此外，开始菜单左侧的“应用分类”下已经是包含了系统中的所有应用，但是在开始菜单右侧却再次突出“所有应用”，这样的设计难免冗余，建议将开始菜单右侧的应用展示方式调整为“常用应用”或“收藏应用”，希望官方改进！

**Comment by BLumia** [comment]
cc @meiyixiang
